annonatate/image.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import re
+from iiif_prezi.factory import ManifestFactory
+import requests
+
+import pdb
+
+class Image:
+    def __init__(self, request_form, request_files, origin_url):
+        self.iiifimage = request_form['upload'] # might be 'uploadimage', or a url
+        self.uploadurl = False
```

Comment:
Not sure why this is here. Also it is removed from original function and it might cause problems.

Reply:
removed line 13